docs/user-guide/examples.md

```diff
@@ -194,6 +190,42 @@ These variables have to be set on the machine/container which host Træfik.
 
 These variables are described [in this section](/configuration/acme/#provider).
 
+### DNS challenge with wildcard domains
+
+```toml
+[entryPoints]
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+
+[acme]
+email = "test@traefik.io"
+storage = "acme.json"
+caServer = "http://172.18.0.1:4000/directory"
+entryPoint = "https"
+  [acme.dnsChallenge]
+  provider = "digitalocean" # DNS Provider name (cloudflare, OVH, gandi...)
+  delayBeforeCheck = 0
+
+[[acme.domains]]
+  main = "*.local1.com"
+  sans = ["test1.local1.com", "test2.local1.com"]
```

Comment:
You can't do that (wildcard with sans) ?

Reply:
No you can't do it for the moment!

Reply:
Really ?